server_test.go

```diff
@@ -280,29 +283,45 @@ func TestLoadConfig(t *testing.T) {
 			},
 		},
 		{
-			file:        "fixtures/config_no_endpoints.yml",
+			file:        filepath.Join("fixtures", "config_no_endpoints.yml"),
+			enabled:     true,
 			shouldFail:  true,
 			errContains: "no CrateDB endpoints provided",
 		},
 		{
-			file:        "fixtures/config_unknown_fields.yml",
+			file:        filepath.Join("fixtures", "config_unknown_fields.yml"),
+			enabled:     true,
 			shouldFail:  true,
 			errContains: "field unknown_fields not found",
 		},
 		{
-			file:        "fixtures/config_invalid_yaml.yml",
+			file:        filepath.Join("fixtures", "config_invalid_yaml.yml"),
+			enabled:     true,
 			shouldFail:  true,
 			errContains: "error unmarshaling YAML",
 		},
 		{
-			file:        "fixtures/config_missing_file.yml",
+			file:        filepath.Join("fixtures", "config_missing_file.yml"),
+			enabled:     runtime.GOOS != "windows",
 			shouldFail:  true,
 			errContains: "no such file or directory",
 		},
+		{
+			file:        filepath.Join("fixtures", "config_missing_file.yml"),
+			enabled:     runtime.GOOS == "windows",
+			shouldFail:  true,
+			errContains: "The system cannot find the file specified",
```

Comment:
Have you thought about making the `errContains` value conditional, instead of adding a new `enabled` field to the struct? It might be a bit more concise.

```suggestion
			errContains: func(os string) string {
				if os == "windows" {
					return "The system cannot find the file specified"
				} else {
					return "no such file or directory"
				}
			}(runtime.GOOS),

```

Reply:
Thank you. I implemented it like that with 49b09f833.